src/Admin/Vue/PaymentMethods.php

```diff
@@ -15,7 +15,7 @@ public function payment_method_standard( ) {
 
 
 		$method = [
-			"redirect" => false,
```

Comment:
you already have the default value above:

switch($option){
			case "popup" : $method["popup"] = true;break;
			case "integrated" : $method["integrated"] = true;break;
			default: $method["redirect"] = true;break;
		}

Reply:
i dont need just a default, i need to force it everytime i switch mode: live/test